creator-node/src/snapbackSM.js

```diff
@@ -449,71 +449,67 @@ class SnapbackSM {
   }
 
   /**
```

Comment:
this function diff is pretty messy, might be easier to view without side-by-side